File: leetsync/3194-find-words-containing-character/find-words-containing-character.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> result = new ArrayList<>();
        int j = 0;
        for (String word : words) {
            int n = word.length();
            for (int i = 0; i < n; i++) {
                if (word.charAt(i) == x) {
                    result.add(j);
                    break;
                }
            }
            j++;
        }
        return result;
    }
}

File: leetsync/3194-find-words-containing-character/find-words-containing-character.go
func findWordsContaining(words []string, x byte) []int {
	result := make([]int, 0, len(words))
	j := 0
	for _, word := range words {
		n := len(word)
		for i := 0; i < n; i++ {
			if word[i] == x {
				result = append(result, j)
				break
			}
		}
		j++
	}
	return result
}